tests/helpers/test_update_coordinator.py

```diff
@@ -73,23 +77,23 @@ def update_callback():
     await crd.async_refresh()
 
     assert crd.data is None
-    assert crd.failed_last_update is True
+    assert crd.last_update_success is False
     assert "Error fetching test data" in caplog.text
 
     crd.update_method = CoroutineMock(return_value=1)
 
     await crd.async_refresh()
 
     assert crd.data == 1
-    assert crd.failed_last_update is False
+    assert crd.last_update_success is True
 
     crd.update_method = CoroutineMock(side_effect=ValueError)
     caplog.clear()
 
     await crd.async_refresh()
 
     assert crd.data == 1  # value from previous fetch
-    assert crd.failed_last_update is True
+    assert crd.last_update_success is False
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4959800)